test/ng/locationSpec.js

```diff
@@ -2480,6 +2480,10 @@ describe('$location', function() {
       expect(parseLinkAndReturn(locationUrl, 'someIgnoredAbsoluteHref', '#test')).toEqual('http://server/pre/otherPath#test');
     });
 
+    it('should cope with double slashes in the path', function() {
+      expect(parseLinkAndReturn(locationUrl, 'http://server/pre///other/path')).toEqual('http://server/pre///other/path');
```

Comment:
I'll look into it...

Krzysztof also suggested that we need to deal with multiple leading backslashes too...